src/pages/Login/Signup.styles.scss

```diff
@@ -0,0 +1,364 @@
+@import url('https://fonts.googleapis.com/css?family=Biryani&display=swap');
+
+body {
+    background: #050516;
```

Comment:
Don't use Body selector here. Try to avoid writing global scss, and write it only in index.scss file.